app/templates/board.html

```diff
@@ -16,10 +19,10 @@
         </div>
     </div>
     <!-- Assignee -->
-    <div class="col-sm-{{ (12/board_sections.length) | number:0 }} bz-column" ng-repeat="board in board_sections" ng-show="ShowByAssignee">
```

Comment:
Ideally, using the same html for both assignee and product view would be best. It would save duplicate code. But this works for now!